app.py

```diff
@@ -141,6 +146,54 @@ def get_user():
     
     return render_template('user-detail.html', form=form)
 
+@app.route('/forgot_password', methods=['GET', 'POST'])
+def forgot_password():
+
+    form = ForgotPasswordForm()
+    if request.method == 'POST':
+        email = request.form['email']
+
+        user = User.query.filter_by(email=email).first()
+        if user:
+            token = generate_reset_token(user)
+
+            msg = Message('Hello, Here is your reset token', sender=mail_username, recipients=[user.email])
+            msg.body = f"http://127.0.0.1:5000/reset_password/{token}"
```

Comment:
This URL needs to be based on the environment

Reply:
done